### DISCUSSION TITLE
Add support for notifications via Telegram

### DISCUSSION THREAD

**Discussion by wtfpeter** [upptime/upptime#88] [Ideas] [answered]
Hi, I'd like to see support for notifications via Telegram bot API in Upptime. Telegram is a messaging platform which provides an API to its users.

Two pieces of information would be required to be provided by the user, preferably in form of Github secrets
- A bot token (generated via `@botfather` on Telegram)
- User ID or group ID to be notified (can be found using `@get_id_bot` on Telegram)

The CI notify action could then trigger a request towards the bot api, using this [method ](https://core.telegram.org/bots/api#sendmessage). 

I saw you are using python requests, see these links for reference
https://stackoverflow.com/a/31198286
https://medium.com/@ManHay_Hong/how-to-create-a-telegram-bot-and-send-messages-with-python-4cf314d9fa3e

**Comment by AnandChowdhary** [comment, marked as answer]
This is added! 😄

https://upptime.js.org/docs/notifications#telegram

**Comment by wtfpeter** [comment]
Thanks, that was quick! The docs currently are refering to using your Telegram username but it probably should be the user ID.
